packages/toast/src/InternalToast/InternalToast.styles.ts

```diff
@@ -12,37 +11,38 @@ import {
 } from '@leafygreen-ui/tokens';
 import { anchorClassName } from '@leafygreen-ui/typography';
 
-import { Variant } from './Toast.types';
+import { TOAST } from '../constants';
```

Comment:
nit: I was kinda confused when I saw this import statement because I didn't know what `TOAST` meant - is it a component? or an instance? Why is it all in caps? Maybe a name like `ToastConstants` is better

Reply:
I think the all-caps make sense, but I agree that the name can be confusing.